core/point/point.go

```diff
@@ -46,55 +43,46 @@ func New(x, y int64) *Point {
 	}
 }
 
-// X returns the x-value.
+// X method receives a pointer argument of type *Point,
+// and returns the x-value.
 func (pt *Point) X() int64 { return pt.x }
 
-// Y returns the y-value.
+// Y method receives a pointer argument of type *Point,
+// and returns the y-value.
 func (pt *Point) Y() int64 { return pt.y }
 
-// ToSGF converts a pointer-type (immutable) *Point
+// ToSGF method receives a pointer argument of type *Point,
+// and converts the pointer-type (immutable) *Point
 // to an SGF Point (two letter string).
-// The returned value is 0-indexed.
 func (pt *Point) ToSGF() (string, error) {
-	var sgfOut string
-	var err01 = fmt.Errorf("")
-	if (pt.X() <= 51) && (pt.Y() <= 51) {
-		sgfX := string(pointToSgfMap[pt.X()])
-		sgfY := string(pointToSgfMap[pt.Y()])
-		sgfOut = sgfX + sgfY
-		err01 = nil
-	} else {
-		sgfOut = ""
-		err01 = fmt.Errorf("*Point int64 x and y value entries must" +
-			" be greater than or equal to 0, " +
-			"and less than or equal to 51. ")
-		// err01 = errors.New("*Point x and y value entries must be" +
-		// 	" greater than or equal to 0, and less than or equal to 51")
+	if (pt.X() < 0) || (51 < pt.X()) || (pt.Y() < 0) || 51 < (pt.Y()) {
+		return "", fmt.Errorf(
+			"*Point int64 x and y value entries must be greater than" +
+				" or equal to 0, " +
+				"and less than or equal to 51. ")
 	}
-	return sgfOut, err01
+	return string(pointToSgfRef[pt.X()]) + string(
+		pointToSgfRef[pt.Y()]), nil
+
 }
 
-// String() method to represent and print a Point, useful for debugging and test purposes
+// String method receives a pointer argument of type Point,
```

Comment:
Again, a bit too wordy. You don't need to describe what's already clear from the type signature.